src/main/java/ch/njol/skript/bukkitutil/SoundUtils.java

```diff
@@ -25,19 +26,17 @@ public final class SoundUtils {
 		soundString = soundString.toUpperCase(Locale.ENGLISH);
 		if (SOUND_IS_INTERFACE) {
 			try {
-				//noinspection deprecation
+				//noinspection deprecation,removal
 				return Sound.valueOf(soundString).getKey();
-			} catch (IllegalArgumentException ignore) {
-			}
+			} catch (Exception ignored) {}
 		} else {
 			try {
 				//noinspection unchecked,rawtypes
 				Enum soundEnum = Enum.valueOf((Class) Sound.class, soundString);
 				return ((Keyed) soundEnum).getKey();
-			} catch (IllegalArgumentException ignore) {
-			}
+			} catch (IllegalArgumentException ignored) {}
 		}
-		return null;
+		return NamespacedKey.fromString(soundString.toLowerCase(Locale.ENGLISH));
```

Comment:
Correct me if I'm wrong, but won't this need a try/catch?
I'm going to assume if an invalid key is entered this will error.

Reply:
It doesnt throw an exception, just returns null

Reply:
Correction, throws exception is the string is null